cryptofeed/pairs.py

```diff
@@ -160,6 +160,9 @@ def coinbene_pairs():
     r = requests.get('http://api.coinbene.com/v1/market/symbol').json()
     return {f"{e['baseAsset']}-{e['quoteAsset']}" : e['ticker'] for e in r['symbol']}
 
+def deribit_pairs():
+    return 'BTC-PERPETUAL'
```

Comment:
these should be dynamically generated, like the others. deribit has a rest endpoint where the specify the supported currencies (and they support more than just BTC)